tools/depends/target/ffmpeg/Makefile

```diff
@@ -26,7 +26,10 @@ ifeq ($(CROSS_COMPILING), yes)
 endif
 ifeq ($(OS), linux)
   ffmpg_config += --target-os=$(OS) --cpu=$(CPU)
-  ffmpg_config += --enable-vdpau --enable-vaapi --enable-pic
+  ifneq ($(TARGET_PLATFORM),raspberry-pi)
+    ffmpg_config += --enable-vdpau
+  endif
+  ffmpg_config += --enable-vaapi --enable-pic
```

Comment:
ffmpeg have changed the configure script since version 3.1
they just ignored the options if the needed libraries couldn't be found